core/trino-main/src/main/java/io/trino/sql/analyzer/Analysis.java

```diff
@@ -957,6 +960,38 @@ public void addColumnMask(Table table, String column, Expression mask)
                 .collect(toImmutableList());
     }
 
+    public void addOriginColumnDetail(Field field, Collection<ColumnEntry> columnEntries)
+    {
+        ImmutableSet.Builder<ColumnEntry> columnEntryBuilder = ImmutableSet.builder();
```

Comment:
names `columnEntryBuilder` and `columnEntries` are not good, it is not easy to see the difference and decide which one should be used.